platform/fetch-native/src/index.ts

```diff
@@ -0,0 +1,8 @@
+import { configureFetch } from '@affinidi/platform-fetch'
+
+export const useNativeFetch = () => {
+  // TODO: fix the types
```

Comment:
Is this still TODO?

Reply:
This was originally intended to be a quick fix for a critical bug, but apparently nobody is interested in it enough, so I'll try to spend some more time on solving all TODOs.

Reply:
Fixed